src/android/DirectoryManager.java

```diff
@@ -56,45 +56,46 @@ public static boolean testFileExists(String name) {
     }
 
     /**
-     * Get the free disk space
-     * 
+     * Get the free space in external storage
+     *
      * @return 		Size in KB or -1 if not available
```

Comment:
Hah, I actually didn't know that the tradition has been for KB to mean 1024. I had someone drill KiB into me recently, so I was under the impression that KiB were the way to go. Thanks for disillusioning me!